ubuntu/standard/Dockerfile.template

```diff
@@ -149,3 +154,21 @@ RUN apt-get update \
  && apt-get install -y --no-install-recommends \
     subversion \
  && rm -rf /var/lib/apt/lists/*
+
+# Download hosted tool cache
+ENV AGENT_TOOLSDIRECTORY=/opt/hostedtoolcache
+RUN azcopy --recursive --source https://vstsagenttools.blob.core.windows.net/tools/hostedtoolcache/linux --destination $AGENT_TOOLSDIRECTORY
+
+# Install the tools from the hosted tool cache
+RUN cd $AGENT_TOOLSDIRECTORY \
```

Comment:
This command will leave the container inside the agent tools directory for startup. This will then fail when it tries to run the startup command "./start.sh" which assumes the container is currently in the /vsts directory. You should be able to use pushd/popd to move into and out of directories temporarily.

Reply:
I wanted to use `pushd` and `popd`, but they are Bash builtins.  We are running in sh here so they are not available.  I did test this container and it still worked, but your argument makes sense.  I can make it so it `cd`s back to the original working directory.